lib/pactflow/client/provider_contracts/publish.rb

```diff
@@ -23,7 +23,9 @@ def initialize(params, options, pact_broker_client_options)
         def do_call
           create_branch_version_and_tags
           create_contract
-          PactBroker::Client::CommandResult.new(true, green("Successfully published provider contract for #{provider_name} version #{provider_version_number} to Pactflow"))
+          formatted_contract_path = "#{pact_broker_base_url}/contracts/bi-directional/provider/#{provider_name}/version/#{provider_version_number}/provider-contract"
```

Comment:
Printing out the UI link is a good idea, but I'll update the code to return a link to print, rather than hardcoding it in the client.

Reply:
We're going to be completely redesigning the UI next year, and the URLs will likely change.